cms/templates/blog/bloglist.html

```diff
@@ -0,0 +1,53 @@
+{% extends "catalogue/base.html" %}
+{% load bulma_tags %}
+{% load field_helpers %}
+{% block title %}{{ component.name }}  - {{ block.super }}{% endblock %}
+{% block body %}
+<section class="section has-background-primary has-text-white">
+    <div class="container is-max-widescreen">
+        <div class="media">
+            <div class="media-content">
+              <h1 class="title is-2 has-text-white">
+                IIP-Ecosphere Erfolgsgeschichten
+              </h1>
+            </div>
+        </div>
+    </div>
+  </section>
+  <section class="section">
+    <div class="container is-max-widescreen">
+        {% if blog_posts|length == 0 %}
+            <p>Noch keine Einträge vorhanden :(</p>
+        {% endif %}
+        <div class="row">
+            {% for post in blog_posts %}
+                <div class="card mb-4">
+                    <div class="card-content">
+                        <p class="title">{{ post.title }}</p>
+                        <p class="subtitle">{{ post.author }} | {{ post.created}} </p>
+                        <div class="content">{{ post.content|safe|slice:":600" }}</div>
+                    </div>
+                    <footer class="card-footer">
+                        <p class="card-footer-item">
+                          <span>
+                            <a href="{% url 'cms:blog_page' post.slug %}">Weiterlesen</a>
```

Comment:
post.get_absolute_url didn't work here for me.

Reply:
because of the typo ;)